src/controller/postUrl.ts

```diff
@@ -0,0 +1,43 @@
+import express from "express";
+import { databaseModel } from "../model";
+import { nanoid } from "nanoid";
+import { logger } from "../services";
+import { response } from "../utils";
+
+export const postUrl = async (req: express.Request, res: express.Response) => {
+  try {
+    const { fullUrl, customUrl } = req.body;
+
+    logger.info("Request body:", req.body);
+
+    if (!fullUrl) {
+      return res.status(400).json(response.error("fullUrl is required."));
+    }
```

Comment:
This check should be in a middleware. Use [express-validator](https://express-validator.github.io/docs/) or [joi](https://joi.dev/)